CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
 
 ### Other changes
 
+* The `BottomBannerViewController` now accounts for the safe-area inset correctly, if applicable. ([#1982](https://github.com/mapbox/mapbox-navigation-ios/pull/1982))
```

Comment:
This is in the v0.29.0 release notes, which is a couple versions behind. Mind moving it up to a new “master” section?